content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,108 @@
+enable our users to utilize Istio's capabilities with confidence, which was another focus for the 1.9 release.
+
+Keep an eye on our blog for our 2021 roadmap, where we will demonstrate our focus on continued improvement in the Day 2
+experience.
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.9.0 is officially supported on Kubernetes versions `1.17.0` to `1.20.x`.
+{{< /tip >}}
+
+Thank you to our users who participated in user experience surveys and empathy sessions, to help us ensure Istio 1.9 is
+our most stable release to date.
+
+This is the first Istio release for 2021.  We would like to thank the entire Istio community, and especially the release
+managers [Shamsher Ansari](https://github.com/shamsher31) (Red Hat), [Steven Landlow](https://github.com/stevenctl)
+(Google) and [Jacob Delgado](https://github.com/jacob-delgado) (Aspen Mesh) for helping to get Istio 1.9.0
+published.
+
+Here are some highlights for this release:
+
+## Virtual Machine Integration (Beta)
+
+Enabling workloads running in VMs to be part of the Istio service mesh, being able to apply consistent policy, and
+collect telemetry across containers and VMs has always been a focus of the Istio community.  We have continued improving
+the stability, testing and documentation for VM integration, and are happy to announce that in Istio 1.9 we have
+promoted this feature to Beta.
+
+Here's a list of supporting documents which you can follow to easily expand your Istio service mesh to include VMs:
+
+* [Virtual Machine Installation](/docs/setup/install/virtual-machine/) to get started.
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
+* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
+* [Bookinfo with a Virtual Machine](/docs/examples/virtual-machines/) to learn more about connecting virtual machine workloads to Kubernetes workloads.
+
+## Request Classification (Beta)
+
+Istio continues to make mesh telemetry collection more configurable. In this release,
+[Request Classification](/docs/tasks/observability/metrics/classify-metrics/) has been promoted to Beta. This feature
+enables users to more precisely understand and monitor the traffic in their service mesh.
+
+## Kubernetes Service API support (Alpha)
+
+Support for [Kubernetes service-apis](https://kubernetes-sigs.github.io/service-apis/) has been an active area of
+development since Istio 1.6, and we are pleased to announce support for them as Alpha in 1.9. Users will benefit in
+adoption of these APIs as a key benefit would allow users to move between service meshes that have adopted the use of
+them. To try them out, check out the [Service API's getting started documentation](/docs/tasks/traffic-management/ingress/service-apis/).
```

Comment:
```suggestion
Configuring Istio to expose a service using [Kubernetes Service APIs](https://kubernetes-sigs.github.io/service-apis/) has been an active area of development since Istio 1.6, and we are pleased to announce support for them as Alpha in 1.9. Using these APIs benefits users who move between other service meshes that support these APIs. To try them out, check out the [Service API's getting started documentation](/docs/tasks/traffic-management/ingress/service-apis/).
```